rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1_src.json

```diff
@@ -329,6 +329,9 @@
                 ],
                 "default": "person",
                 "form": {
+                  "templateOptions": {
+                    "cssClass": "col-lg-6"
```

Comment:
Really a small detail, but it is a CSS class, or an HTML one? I remember the old angular form library was calling these `htmlClass`.

Reply:
What's the difference between CSS and HTML classes actually?

Reply:
Ok, sorry for disturbing on such a thing. In my understanding, this is something I would see in a Jinja2 template in the HTML, but you're right, on the bootstrap side, it's also in the CSS, I forgot.

So leave it as it is.